artemis-core-client/src/main/java/org/apache/activemq/artemis/api/config/ActiveMQDefaultConfiguration.java

```diff
@@ -429,6 +429,12 @@ public static String getDefaultHapolicyBackupStrategy() {
    // Default period to wait between configuration file checks
    public static final long DEFAULT_CONFIGURATION_FILE_REFRESH_PERIOD = 5000;
 
+   public static final long DEFAULT_GLOBAL_MAX_SIZE = -1;
```

Comment:
USAGE implies %, and it would be confusing with the other parameter.

AddressSize's counterpart of this is called getMaxSizeBytes. I could call rename them all to maxSizeBytes. (I was happy with just max_size though :) )